src/main/java/net/mcreator/ui/workspace/resources/TextureType.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Arrays;
 
 public enum TextureType {
-	BLOCK("block"), ITEM("item"), ARMOR("armor"), OTHER("other");
+	BLOCK("block"), ITEM("item"), ARMOR("armor"), VILLAGER_PROFESSION("villager_profession"), OTHER("other");

```

Comment:
Is a new texture type really needed? We should try to avoid as much as possible adding new texture types when they are not needed. Custom villagers will not be used as much as blocks or entities. I agree not every misc texture should go in `OTHERS`. This is why we should try to create as much general categories as possible. For the moment, VP should go in Others and be moved the same way paintings are made. When https://github.com/MCreator/MCreator/pull/2537 will be merged, we can then move VP to entities and move them to their specific folder.